src/cmd/services/m3aggregator/config/aggregator.go

```diff
@@ -779,3 +778,91 @@ func setMetricPrefix(
 	}
 	return fn([]byte(*str))
 }
+
+// passThroughWriter writes passthrough metrics to backends.
+type passThroughWriter struct {
+	numShards int
+	writers   []writer.Writer
+	locks     []sync.Mutex
+	shardFn   sharding.ShardFn
+}
+
+func (c *AggregatorConfiguration) newPassThroughWriter(
+	cs client.Client,
+	iOpts instrument.Options,
+	scope tally.Scope,
+	shardFn sharding.ShardFn,
+) (writer.Writer, error) {
+	// This is a temporary change to use a separate m3msg topic for pass-through metrics during the migration.
```

Comment:
Can this comment be marked with a `TODO` so that it can be found later for removal? Not sure if this will be needed permanently for all users since we don't know the progression of migrations for all users?